From 781fba42333635a520cbd947a58ca6a628cd9eec Mon Sep 17 00:00:00 2001
From: demorripv <iaideya@clemson.edu>
Date: Fri, 25 Apr 2025 12:20:18 -0400
Subject: [PATCH] added  2nd 5th s line

---
 src/main.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/main.c b/src/main.c
index 53a91fc..7ee1b17 100644
--- a/src/main.c
+++ b/src/main.c
@@ -121,7 +121,7 @@ int main(void)
 
 /*2
 	 * Infinite 
-	 * =======ssss
+	 * =======sssss
 	 **********/
 /*3
 	 * Infinite 
-- 
2.25.1

