From 0fcf33fcb5d49fa93b0b066a9037246461c68b94 Mon Sep 17 00:00:00 2001
From: demorripv <iaideya@clemson.edu>
Date: Fri, 25 Apr 2025 12:25:35 -0400
Subject: [PATCH] added  2nd 7th s line

---
 src/main.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/main.c b/src/main.c
index 65f8f5f..6c4867c 100644
--- a/src/main.c
+++ b/src/main.c
@@ -121,7 +121,7 @@ int main(void)
 
 /*2
 	 * Infinite 
-	 * =======ssssss
+	 * =======sssssss
 	 **********/
 /*3
 	 * Infinite 
-- 
2.25.1

