From f8e809cbba99220d72ecbfaad12828dac059c9cc Mon Sep 17 00:00:00 2001
From: demorripv <iaideya@clemson.edu>
Date: Fri, 25 Apr 2025 13:01:40 -0400
Subject: [PATCH] added  1st 11th s line

---
 src/main.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/main.c b/src/main.c
index 0f5fde7..8239abd 100644
--- a/src/main.c
+++ b/src/main.c
@@ -116,7 +116,7 @@ int main(void)
 }
 /*!
 	 * Infinite 
-	 * =======ssssssssss
+	 * =======sssssssssss
 	 **********/
 
 /*2
-- 
2.25.1

