From 6877aa928c3049a9e0d2e0df462a8b21bc820135 Mon Sep 17 00:00:00 2001
From: demorripv <iaideya@clemson.edu>
Date: Wed, 23 Apr 2025 22:00:03 -0400
Subject: [PATCH] added 3 s line

---
 src/main.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/main.c b/src/main.c
index 528d510..a7df5e1 100644
--- a/src/main.c
+++ b/src/main.c
@@ -125,5 +125,5 @@ int main(void)
 	 **********/
 /*3
 	 * Infinite 
-	 * =======ss
+	 * =======sss
 	 **********/
\ No newline at end of file
-- 
2.25.1

