From 09e609b2b1e499ef6540e7ae372e1c0779229865 Mon Sep 17 00:00:00 2001
From: demorripv <iaideya@clemson.edu>
Date: Fri, 25 Apr 2025 12:22:34 -0400
Subject: [PATCH] added  2nd 6th s line

---
 src/main.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/main.c b/src/main.c
index 7ee1b17..65f8f5f 100644
--- a/src/main.c
+++ b/src/main.c
@@ -121,7 +121,7 @@ int main(void)
 
 /*2
 	 * Infinite 
-	 * =======sssss
+	 * =======ssssss
 	 **********/
 /*3
 	 * Infinite 
-- 
2.25.1

