From 4f3bd3ad337bd7615be517c3b081b039aa48b1ab Mon Sep 17 00:00:00 2001
From: demorripv <iaideya@clemson.edu>
Date: Fri, 25 Apr 2025 12:15:31 -0400
Subject: [PATCH] added  2nd 3th s line

---
 src/main.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/main.c b/src/main.c
index 3049257..4a0a4bb 100644
--- a/src/main.c
+++ b/src/main.c
@@ -121,7 +121,7 @@ int main(void)
 
 /*2
 	 * Infinite 
-	 * =======ss
+	 * =======sss
 	 **********/
 /*3
 	 * Infinite 
-- 
2.25.1

