From 2dd9197c838418fd6f6468d9d84e37660cb667c3 Mon Sep 17 00:00:00 2001
From: demorripv <iaideya@clemson.edu>
Date: Fri, 25 Apr 2025 12:39:35 -0400
Subject: [PATCH] added  2nd 12th s line

---
 src/main.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/main.c b/src/main.c
index b0742b4..9a85c8a 100644
--- a/src/main.c
+++ b/src/main.c
@@ -121,7 +121,7 @@ int main(void)
 
 /*2
 	 * Infinite 
-	 * =======sssssssssss
+	 * =======ssssssssssss
 	 **********/
 /*3
 	 * Infinite 
-- 
2.25.1

