From 611f37097893dacf806b4e9c386b3c007e45939c Mon Sep 17 00:00:00 2001
From: demorripv <iaideya@clemson.edu>
Date: Fri, 25 Apr 2025 11:45:49 -0400
Subject: [PATCH] added 9th s line

---
 src/main.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/main.c b/src/main.c
index a978936..7c217d1 100644
--- a/src/main.c
+++ b/src/main.c
@@ -125,5 +125,5 @@ int main(void)
 	 **********/
 /*3
 	 * Infinite 
-	 * =======ssssssss
+	 * =======sssssssss
 	 **********/
\ No newline at end of file
-- 
2.25.1

